repository/w.json

```
@@ -901,13 +901,13 @@
 			]
 		},
 		{
-			"name": "World of Warcraft TOC file Syntax",
-			"details": "https://github.com/AndrielChaoti/wowtoc-syntax",
+			"name": "World of Warcraft TOC",
+			"details": "https://github.com/p3lim/sublime-toc",
```

Since you're changing the name (good 👍🏻 ) you need to add previous_names:

```suggestion
			"previous_names": ["World of Warcraft TOC file Syntax"],
			"details": "https://github.com/p3lim/sublime-toc",
```